packages/mui-base/src/unstable_useNumberInput/useNumberInput.test.tsx

```diff
@@ -8,6 +8,10 @@ import {
   UseNumberInputParameters,
 } from '@mui/base/unstable_useNumberInput';
 
+// TODO for v6 initialize @testing-library/user-event using userEvent.setup() instead of directly calling methods e.g. userEvent.click() for all related tests in this file
```

Comment:
```suggestion
// TODO v6: Initialize @testing-library/user-event using userEvent.setup() instead of directly calling methods e.g. userEvent.click() for all related tests in this file
```